app/src/main/java/com/example/rajadav/adidas/DataAdapter.java

```diff
@@ -0,0 +1,73 @@
+package com.example.rajadav.adidas;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.example.rajadav.adidas.database.Goal;
+
+import java.util.List;
+
+
+
+public class DataAdapter extends RecyclerView.Adapter<DataAdapter.DataAdapterViewHolder> {
```

Comment:
Let'a start the good practices. Add some description at the beginning of this class saying what is doing. In this case, it is an adapter, which is a well known element for other devs, so a few courtesy words will do the work, but for other classes that you create might be interesting to add more.